README.md

```diff
@@ -15,9 +15,18 @@ CloudAI operates on four main schemas:
 These schemas enable CloudAI to be flexible and compatible with different systems and configurations.
 
 ## Quick Start
-First, ensure you have access to the docker repository. This is a manual step:
-- In https://ngc.nvidia.com/signin go to setup -> generate api key
-- Set up enroot credentials properly. Ensure you have the correct credentials under `~/.config/enroot/.credentials`:
+First, ensure you have access to the Docker repository. Follow these steps:
+
+1. **Sign In**: Go to [NVIDIA NGC](https://ngc.nvidia.com/signin) and sign in with your credentials.
+2. **Join the Correct Group**: Ensure you are part of the `CloudAI/cloudai` group. This will give you access to the necessary Docker images.
```

Comment:
Do we plan to make the images public right away? 

Should we add an alternative path to using existing PyTorch images directly?

Reply:
We can wait until it becomes public. I wanted to add more detailed steps for setting up API keys for users.

Reply:
AFAIK, for public images no setup is needed. For example, images under `nvidia` org are available without any auth.

Maybe we shouldn't mention `Cloudai/cloudai` here yet, but otherwise it is very useful information.

Reply:
Updated, @amaslenn & @srinivas212.